### ISSUE TITLE
Building a classifier and calculating accuracy

### ISSUE THREAD

**Issue by megharana** [megharana/mmt-challange#10]
1. Use TensorFlow to Construct a Neural Network Classifier.
2. Use the UCI Data to Train the Neural Network.
3. Determine the Accuracy of our Neural Network Model.

**Comment by megharana** [comment]
Completed
